internal/ssm/install.go

```diff
@@ -18,37 +21,99 @@ import (
 	"github.com/aws/eks-hybrid/internal/util/cmd"
 )
 
-const (
-	installerPath = "/opt/ssm/ssm-setup-cli"
+// TODO: a better way to override this
+var (
+	InstallerPath = "/opt/ssm/ssm-setup-cli"
```

Comment:
i think there are two ways do avoid this
* make the path configurable in the function we call: either adding some kind of options to the install method (variadic clousures?) or making this function a method and configuring the path as a member of the receiver 
* Create a new method `InstallTo` that takes a path. From the tests we would test this method, and we would change `Install` to call `Install(..., installerPath)`. This doesn't test "all the code" but hides details from the caller if it doesn't need to configure the path.

I think i lean towards the latter but your call